### PR TITLE
[Examples] Add .next to .gitignore

### PR DESCRIPTION
**Pull request by amorriscode**
The TypeScript blog starter didn't have `.next` in the `.gitignore` file.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 11.9s | 11.6s | -217ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.918 | 1.893 | -0.02 |
| / avg req/sec | 1303.13 | 1320.83 | +17.7 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.144 | 1.139 | 0 |
| /error-in-render avg req/sec | 2185.94 | 2195.09 | +9.15 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..e4a8.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.72 kB | 5.72 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 953 B | 953 B | ✓ |
| link.html gzip | 961 B | 961 B | ✓ |
| withRouter.html gzip | 945 B | 945 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.1s | 11.6s | -514ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..e4a8.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.72 kB | 5.72 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 876 kB | 876 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 877 kB | 877 kB | ✓ |
| link.js | 916 kB | 916 kB | ✓ |
| routerDirect.js | 910 kB | 910 kB | ✓ |
| withRouter.js | 910 kB | 910 kB | ✓ |
| Overall change | 4.5 MB | 4.5 MB | ✓ |

</details>

</details>
Commit: 61aa66c1729e40a5706a56f7d86ff283749e5461

File: examples/blog-starter-typescript/.gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .env
-.now
\ No newline at end of file
+.now
+.next
```
Comment (lfades) [inline examples/blog-starter-typescript/.gitignore]:
```suggestion
# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.

# dependencies
/node_modules

# testing
/coverage

# Next.js
/.next/
/out/

# production
/build

# misc
.DS_Store

# debug
npm-debug.log*
yarn-debug.log*
yarn-error.log*

# local env files
.env.local
.env.development.local
.env.test.local
.env.production.local

# Vercel
.vercel
```

Related to https://github.com/vercel/next.js/issues/15102

Comment (lfades) [inline examples/blog-starter-typescript/.gitignore]: Related to https://github.com/vercel/next.js/issues/15102

```suggestion
# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
# dependencies
/node_modules
# testing
/coverage
# Next.js
/.next/
/out/
# production
/build
# misc
.DS_Store
# debug
npm-debug.log*
yarn-debug.log*
yarn-error.log*
# local env files
.env.local
.env.development.local
.env.test.local
.env.production.local
# Vercel
.vercel
```

Comment (lfades) [inline examples/blog-starter-typescript/.gitignore]:
```suggestion
# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
# dependencies
/node_modules
# testing
/coverage
# Next.js
/.next/
/out/
# production
/build
# misc
.DS_Store
# debug
npm-debug.log*
yarn-debug.log*
yarn-error.log*
# local env files
.env.local
.env.development.local
.env.test.local
.env.production.local
# Vercel
.vercel
```

**Review by lfades — APPROVED** [review]
@amorriscode Thank you!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.6s | -137ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.219 | 2.265 | ⚠️ +0.05 |
| / avg req/sec | 1126.81 | 1103.55 | ⚠️ -23.26 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.46 | 1.537 | ⚠️ +0.08 |
| /error-in-render avg req/sec | 1712.08 | 1626.6 | ⚠️ -85.48 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..e4a8.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.72 kB | 5.72 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 953 B | 953 B | ✓ |
| link.html gzip | 961 B | 961 B | ✓ |
| withRouter.html gzip | 945 B | 945 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.5s | 13.6s | ⚠️ +60ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..e4a8.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.72 kB | 5.72 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 878 kB | 878 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 878 kB | 878 kB | ✓ |
| link.js | 918 kB | 918 kB | ✓ |
| routerDirect.js | 911 kB | 911 kB | ✓ |
| withRouter.js | 911 kB | 911 kB | ✓ |
| Overall change | 4.5 MB | 4.5 MB | ✓ |

</details>

</details>
Commit: cb4a45129251e0fe8fe4c46b2bb01963bbb324b3

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12s | 11.9s | -127ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.106 | 2.047 | -0.06 |
| / avg req/sec | 1187.05 | 1221.52 | +34.47 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.338 | 1.386 | ⚠️ +0.05 |
| /error-in-render avg req/sec | 1869.02 | 1803.11 | ⚠️ -65.91 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..e4a8.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.72 kB | 5.72 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 953 B | 953 B | ✓ |
| link.html gzip | 961 B | 961 B | ✓ |
| withRouter.html gzip | 945 B | 945 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 13.3s | 12.8s | -499ms |
| nodeModulesSize | 66.4 MB | 66.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..e4a8.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.72 kB | 5.72 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | amorriscode/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 878 kB | 878 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 878 kB | 878 kB | ✓ |
| link.js | 918 kB | 918 kB | ✓ |
| routerDirect.js | 911 kB | 911 kB | ✓ |
| withRouter.js | 911 kB | 911 kB | ✓ |
| Overall change | 4.5 MB | 4.5 MB | ✓ |

</details>

</details>
Commit: e422d1e11d152518ac45f45fda399c49a2b39143

**Pull request merged**